lib/ansible/playbook/base.py

```diff
@@ -112,6 +112,9 @@ def _generic_g(prop_name, self):
         if hasattr(self, method):
             return getattr(self, method)()
 
+        if prop_name not in self._attributes:
+            return None
+
```

Comment:
@abadger The behavior of hasattr changed in Python 3.2 (http://stackoverflow.com/questions/903130/hasattr-vs-try-except-block-to-deal-with-non-existent-attributes/16186050#16186050)  so it only catches AttributeError.

We may want to solve this a different way, but without this an exception will be raised when prop_name doesn't exist under Python 3.